rehive/api/resources/auth_resources.py

```diff
@@ -167,7 +167,22 @@ def get_resource_name(cls):
         return 'tokens'
 
 
-class APIAuthMFA(Resource):
+class APIAuthenticators(ResourceList):
+    def create(self, type, details, **kwargs):
+        return super().create(type=type, details=details, **kwargs)
+
+    @classmethod
+    def get_resource_name(cls):
+        return 'authenticators'
+
+
+class APIAuthMFA(Resource, ResourceCollection):
+    def __init__(self, client, endpoint='', filters=None):
+        self.resources = (APIAuthenticators,)
+
+        super().__init__(client, endpoint, filters)
+
+        self.create_resources(self.resources)
```

Comment:
Please remove the `enable_sms()`, `authorize_token()` and `verify()` method on line 187, 193, 196. These are no longer required with the new `deliver()` and `verify()` methods.